### PR TITLE
@W-9790306@ Custom templates config

### PR DESCRIPTION
**Pull request by xyc**
See [Support Custom Templates in CLI](https://salesforce.quip.com/GA1rASFbBX30)

### CODE REVIEW

File: messages/config.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "InvalidConfigValue": "Invalid config value. %s",
   "InvalidInstanceUrl": "Specify a valid Salesforce instance URL",
   "InvalidApiVersion": "Specify a valid Salesforce API version, for example, 42.0",
+  "InvalidCustomOrgMetadataTemplates": "Specify a valid Custom Org Metadata Templates repository URL or location",
```
Comment (jshackell-sfdc) [inline messages/config.json:6]: As I understand it, this config value will simply point to a directory that contains the templates, right?  i.e. it's NOT going to automatically clone/download a repo?  If so, I suggest this wording:

"Specify a valid directory that contains the custom project templates for working with org metadata."

Okay? @mshanemc , I'd love your opinion on the wording.  For example, is it weird say "for working with org metadata" when there's currently only one type of templates?  This is part of the "thinking aheadd when the CLI works with all clouds", but is it confusing to mention it now?

Comment (jshackell-sfdc) [reply messages/config.json:6]: Also, is there an equivalent environment variable?  If so, what is it? Thx!

Comment (xyc) [reply messages/config.json:6]: > this config value will simply point to a directory that contains the templates, right? i.e. it's NOT going to automatically clone/download a repo
It's not going to automatically clone/download a repo when `config:set` command is run, but you can specify a repo URL as the config value and the client (CLI / VS Code) will download the repo when creating with the custom templates.

Comment (xyc) [reply messages/config.json:6]: The env variable would be automatically applied like other env variables: `SFDX_CUSTOM_ORG_METADATA_TEMPLATES`

Comment (jshackell-sfdc) [reply messages/config.json:6]: Gotcha.  So now I have just a small edit to that error message:

"Specify a valid repository URL or directory for the custom org metadata templates."

Comment (xyc) [reply messages/config.json]: Thanks, updated the message!

**Review by mshanemc — APPROVED** [review]
approved.  Adding Mike to the approvals, too.

**Review by jshackell-sfdc — APPROVED** [review]

**Review by sbudhirajadoc — APPROVED** [review]

**Review by mdonnalley — APPROVED** [review]

**Pull request merged**